assets/html/js/metadata.js

```diff
@@ -4,10 +4,13 @@
 $(document).ready(function () {
   let meta = $("div[data-docstringscollapsed]").data();
 
-  if (meta.docstringscollapsed) {
-    $("#documenter-article-toggle-button").trigger({
-      type: "click",
-      noToggleAnimation: true,
-    });
+  // Check if metadata object exists
+  if (meta) {
+    if (meta.docstringscollapsed) {
```

Comment:
This might be cleaner?

```suggestion
    if (meta?.docstringscollapsed) {
```